composer.json

```diff
@@ -14,7 +14,7 @@
         "php": ">=5.4",
         "php-http/httplug": "^1.1",
         "php-http/message-factory": "^1.0",
-        "php-http/message": "^1.2",
+        "php-http/message": "^1.6@dev",
```

Comment:
see the TODO list @aaa2000 added. i think this is correct to show that the PR will merge once 1.6 is released. its just that we should not merge until 1.6 is released and then we can change this line to remove `@dev` before merging.

changing global minimum stability will not change anything for users of this lib, but would potentially have effects on how we install other dependencies.